packages/admin/src/components/Manager/ScenarioQueue/style.ts

```diff
@@ -0,0 +1,58 @@
+import { css } from "@emotion/css";
+import { colors } from "@shared/styles/color";
+import { hashClassNames } from "src/lib/hash";
+
+export default () => {
+  const { scenarioQueueTitle, queue, prev, next, current } = hashClassNames([
+    "scenarioQueueTitle",
+    "queue",
+    "prev",
+    "next",
+    "current",
+  ]);
+
+  const scenarioQueue = css`
+    display: flex;
+    flex-direction: column;
+    position: relative;
+    width: 32rem;
+    min-height: 16rem;
+    margin: 0 0 2rem 2rem;
```

Comment:
```suggestion
    margin: 0 2rem 2rem 0;
```

마진 때문에 화면 가로폭이 좁아졌을 때 왼쪽 정렬이 안 되는 것 같은데, 혹시 마진 세팅을 이렇게 바꾸는 건 어떨까요?

<img width="554" alt="스크린샷 2022-01-10 오후 4 34 12" src="https://user-images.githubusercontent.com/71015915/148731404-47c225a2-7cba-4111-a7a5-afd8eb2f3519.png">

적용 결과는 이렇습니다~

<img width="459" alt="스크린샷 2022-01-10 오후 4 31 04" src="https://user-images.githubusercontent.com/71015915/148731506-888e1b40-2b4b-4904-a202-424b563b90fe.png">

<img width="724" alt="스크린샷 2022-01-10 오후 4 36 30" src="https://user-images.githubusercontent.com/71015915/148731596-b84f11dd-4d9a-4e48-aa72-9ddd8b60dd28.png">

Reply:
저도 그래서 상당히 거슬렸는데 아무래도 박스 크기가 크다보니 꽉차보여서 이렇게 했었습니다ㅠㅠ
그렇다면 시나리오 큐, 실행 로그 둘다 margin-left를 5rem 정도로 해보는건 어떨까요??!!

Reply:
> 저도 그래서 상당히 거슬렸는데 아무래도 박스 크기가 크다보니 꽉차보여서 이렇게 했었습니다ㅠㅠ 그렇다면 시나리오 큐, 실행 로그 둘다 `margin-left`를 `5rem` 정도로 해보는건 어떨까요??!!

그것도 좋은 방법인 것 같아요~ 하지만 카드 디자인이다보니 아무래도 `margin-left` 값은 밑의 시나리오 카드 마진과 통일하는 게 예뻐보일 것 같기도 하네요ㅠ

<img width="454" alt="스크린샷 2022-01-10 오후 7 58 16" src="https://user-images.githubusercontent.com/71015915/148754997-8dd9f455-bbce-4d15-8e9b-81f86cc9ddd5.png">

Reply:
<img width="1792" alt="스크린샷 2022-01-12 오후 10 48 44" src="https://user-images.githubusercontent.com/62797441/149152500-9a42b49c-5d5e-4bd6-ae2d-22180c22538d.png">
<img width="1487" alt="스크린샷 2022-01-12 오후 10 48 55" src="https://user-images.githubusercontent.com/62797441/149152510-c4218a70-b5c2-4255-aa4a-c5e9e3a4a7cc.png">
@soonitoon 이렇게 하는건 어떤가요??

Reply:
좋다고 생각합니다~